pkg/restapi/webhook/webhook.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package webhook
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+)
+
+var logger = log.New("aries-framework/agentd")
+
+type dispatcher struct {
+	client         *http.Client
+	subscriberURLs []string
+}
+
+// StartWebhookDispatcher will launch the webhook dispatcher in a new goroutine.
+// The dispatcher is responsible for sending notifications to subscribers.
+func StartWebhookDispatcher(subscriberURLs []string) {
+	dispatcher := &dispatcher{
+		client:         &http.Client{},
+		subscriberURLs: subscriberURLs,
+	}
+
+	go dispatcher.start()
+}
+
+func (d *dispatcher) start() {
+	ticker := time.NewTicker(2 * time.Second)
+	defer ticker.Stop()
+	for range ticker.C {
+		d.dispatch()
+	}
+}
+
+func (d *dispatcher) dispatch() {
+	for _, subscriberURL := range d.subscriberURLs {
+		go func(destination string) {
+			req, err := http.NewRequest("POST", destination,
+				bytes.NewBufferString(fmt.Sprintf("Sample notification from aries-agentd.")))
```

Comment:
You are not formatting anything here, use simple string

Reply:
Fixed!